cmd/crank/beta/render/runtime.go

```diff
@@ -49,7 +50,7 @@ const (
 // A Runtime runs a Function.
 type Runtime interface {
 	// Start the Function.
-	Start(ctx context.Context) (RuntimeContext, error)
+	Start(ctx context.Context, logger logging.Logger) (RuntimeContext, error)
```

Comment:
```suggestion
	Start(ctx context.Context, l logging.Logger) (RuntimeContext, error)
```

Nit. Or `log` if `l` is too short, but "logger logging logger" stutters a lot. 😄 In other contexts I can live with it because the `logger` variable tends to live for a lot of lines (you can't always easily see what it is from its type). Maybe `log` everywhere would be a good compromise though? 🤔